packages/cli/src/CredentialsHelper.ts

```diff
@@ -257,17 +267,17 @@ export class CredentialsHelper extends ICredentialsHelper {
 			throw new Error(`Credential "${nodeCredential.name}" of type "${type}" has no ID.`);
```

Comment:
Does this error need to be reported, or can we change this as well?

Reply:
Assuming creds with `null` IDs are similar to workflows with `undefined` IDs (this shouldn't happen), then we should keep the guard reporting to Sentry, i.e. same reasoning as in https://github.com/n8n-io/n8n/pull/7535#issuecomment-1782897120